nemo/collections/asr/models/classification_models.py

```diff
@@ -138,9 +151,36 @@ def transcribe(self, paths2audio_files: List[str], batch_size: int = 4, logprobs
             logging.set_verbosity(logging_level)
         return labels
 
+    def prepare_manifest(self, config):
```

Comment:
Add the pytorch lightning decorator to run this **only on device=0**.

Also add documentation. Is this meant to be run prior to train? Or in general? If its general, remove it from Model and put it in a helper script.